deploy/olm-catalog/mattermost-operator/0.5.0/mattermost-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,189 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"mattermost.com/v1alpha1","kind":"ClusterInstallation","metadata":{"name":"example-clusterinstallation"},"spec":{"ingressName":"example.mattermost-example.dev"}}]'
+    capabilities: Basic Install
+  name: mattermost-operator.v0.5.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: ClusterInstallation
+      name: clusterinstallations.mattermost.com
+      version: v1alpha1
+  description: Mattermost is an open source, private cloud and secure collaboration tool
```

Comment:
@jwilander let me know if we need to update this description

Reply:
@wiersgallak this will be the description of Mattermost that will appear on OperatorHub, can you take a look?

Reply:
Here is the tagline we use on the website: Mattermost is a flexible, open source messaging platform 
that enables secure team collaboration

Reply:
```suggestion
  description: Mattermost is a flexible, open source messaging platform that enables secure team collaboration.
```

Reply:
done